README.md

```diff
@@ -49,11 +49,16 @@ scoop install ffmpeg
 - Beam Size: Adjust the amount of transcriptions generated and considered, which affects accuracy and transcription generation time.
 - Transcription Method: Choose "real-time" for real-time diarization and transcriptions, or "sequential" for periodic transcriptions with more context.
 
+## Latency
+If you want minimal latency, use the real-time mode. If you don't mind growing latency and prioritize accuracy, use the sequential mode.
+
 ## Known Bugs
 
-1. On MacOS, there's a clash between av files preventing transcription (works well on Google Colab with Python 3.8).
-2. In the sequential mode, there may be uncontrolled speaker swapping, which can be fixed by using pyannote's building blocks and handling speakers manually.
-3. In real-time mode, audio data not meeting the transcription timeout won't be transcribed.
+1. You may have ffmpeg errors when locally running the application. Create an issue including your operating system and the steps to reproduce.
+2. You may encounter problems when building the wheel for "safetensors". On MacOS, run `brew install rust` to fix it.
```

Comment:
This is not a known issue, but what’s called troubleshooting. FYI for now

Reply:
Do you want me to add a "Troubleshooting" section?